HttpHelpers.cs

```diff
@@ -6,6 +6,18 @@ namespace DUCK.Http
 {
 	public static class HttpHelpers
 	{
+#if UNITY_2017_3_OR_NEWER
```

Comment:
What changed? Does Unity's web request no longer give us a DownloadHandlerTexture?

Also `UnityWebRequestTexture` appears to be in and work the same way in 2017.1 on the docs, can this be made to be compatible all the way down to there?

Reply:
> What changed? Does Unity's web request no longer give us a DownloadHandlerTexture?

Correct, Unity's web request no longer gives us a DownloadHandlerTexture.

> Also UnityWebRequestTexture appears to be in and work the same way in 2017.1 on the docs, can this be made to be compatible all the way down to there?

I thought this too when I visited the page, however if you hover over Other Versions a second time, the values get updated to what I can only assume are the real values as it doesn't change again after that.

Reply:
That's a little frustrating, moving stuff out to the helper like this makes the whole simplification of the process feel a bit more creaky. I wonder how far down the line we get before its actually easier to just use `UnityWebRequest.Get` or `UnityTextureWebRequest.GetTexture`?

Reply:
Yeah as Unity updates there is less of a need for the HttpRequest / HttpResponse classes, I still think the Http class is valid for cutting out the need for MonoBehaviour to Coroutine. There is potential for using UnityWebRequest extension methods for Sending and Aborting the requests which would make HttpRequest and HttpResponse redundant.